src/main/java/net/masterthought/cucumber/ReportInformation.java

```diff
@@ -240,6 +241,16 @@ private void processFeatures() {
             List<ScenarioTag> scenarioList = new ArrayList<>();
             Element[] scenarios = feature.getElements();
             numberOfScenarios = getNumberOfScenarios(scenarios);
+
+            // set filenames for features
+            int count = 1;
+            if (featureNamesCount.containsKey(feature.getFileName())) {
+                count = featureNamesCount.get(feature.getFileName());
+                count++;
+            }
+            featureNamesCount.put(feature.getFileName(),count);
+            feature.setFileName(count);
```

Comment:
file name is used when generating html files so in that case we lost such information and use number values as the file name - is that correct? If so, I don't want to loose such information.